content/pt-br/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -0,0 +1,285 @@
+que permitem acesso externo a serviços em um cluster. Um
+[Ingress controller](/docs/concepts/services-networking/ingress-controllers/)
+cumpre as regras estabelecidas no Ingress.
+
+Essa pagina mostra como configurar um Ingress simples que redireciona as requisições para o Service "web" ou "web2" dependendo do URI HTTP.
+
+## {{% heading "prerequisites" %}}
+
+Esse tutorial assume que você esta usando `minikube` para rodar um cluster Kubernetes local.
+Visite [Install tools](/docs/tasks/tools/#minikube) para aprender como instalar o `minikube`.
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+Se você estiver usando uma versão mais antiga do Kubernetes, veja a documentação para essa versão.
+
+### Criando um cluster minikube
+Se você ainda não configurou um cluster local, rode `minikube start` para criar um cluster.
+
+<!-- steps -->
+
+## Ativando o Ingress controller
+
+1. Para ativar o NGINX Ingress controller, rode os seguintes comandos:
+
+   ```shell
+   minikube addons enable ingress
+   ```
+
+1. Verifique que o NGINX Ingress controller esta rodando
+
+   ```shell
+   kubectl get pods -n ingress-nginx
+   ```
+
+   {{< note >}}
+   Pode demorar alguns minutos antes dos pods estarem rodando OK.
+   {{< /note >}}
+
+   O resultado deve ser similar a:
+
+   ```none
+   NAME                                        READY   STATUS      RESTARTS    AGE
+   ingress-nginx-admission-create-g9g49        0/1     Completed   0          11m
+   ingress-nginx-admission-patch-rqp78         0/1     Completed   1          11m
+   ingress-nginx-controller-59b45fb494-26npt   1/1     Running     0          11m
+   ```
+
+## Deployando uma aplicação hello world
+
+1. Crie um Deployment usando o seguinte comando:
+
+   ```shell
+   kubectl create deployment web --image=gcr.io/google-samples/hello-app:1.0
+   ```
+
+   O resultado deve ser:
+
+   ```none
+   deployment.apps/web created
+   ```
+
+1. Exponha o Deployment:
+
+   ```shell
+   kubectl expose deployment web --type=NodePort --port=8080
+   ```
+
+   O resultado deve ser:
+
+   ```none
+   service/web exposed
+   ```
+
+1. Verifique que o Service esta criado e disponível em uma porta do node:
+
+   ```shell
+   kubectl get service web
+   ```
+
+   O resultado deve ser similar:
+
+   ```none
+   NAME      TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
+   web       NodePort   10.104.133.249   <none>        8080:31637/TCP   12m
+   ```
+
+1. Visite o Service via NodePort:
+
+   ```shell
+   minikube service web --url
+   ```
+
+   O resultado é similar a:
+
+   ```none
+   http://172.17.0.15:31637
+   ```
+
+   ```shell
+   curl http://172.17.0.15:31637 
+   ```
+
+   O resultado é similar a:
+
+   ```none
+   Hello, world!
+   Version: 1.0.0
+   Hostname: web-55b8c6998d-8k564
+   ```
+
+   Você agora pode acessar a aplicação de exemplo através do endereço IP do Minikube e NodePort.
+   O proximo passo te permite acessar a aplicação usando o recurso Ingress.
```

Comment:
You've been using "você" instead of "tu", so for grammatical purposes it would be better to write something like:
```suggestion
   No próximo passo, você irá acessar a aplicação usando o recurso Ingress.
```

"próximo" is also missing an accent in this sentence.